src/test/java/org/takes/rq/RqRequestLineTest.java

```diff
@@ -99,11 +111,16 @@ public void extractsParams() throws IOException {
      * Request without Request-Line.
      * @throws IOException If some problem inside
      */
-    @Test(expected = HttpException.class)
+    @Test
     public void failsOnAbsentRequestLineToken() throws IOException {
-        new RqRequestLine.Base(
-            new RqSimple(Collections.<String>emptyList(), null)
-        ).method();
+        Assertions.assertThrows(
+            HttpException.class,
+            () -> {
+                new RqRequestLine.Base(
+                    new RqSimple(Collections.<String>emptyList(), null)
```

Comment:
@marceloamadeu Use `ListOf` instead `Collections.<String>emptyList()` and remove this ugly null